Extensions/Flows/Startup.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Lombiq.HelpfulExtensions.Extensions.Flows;
+using Lombiq.HelpfulExtensions.Extensions.Flows.Drivers;
+using Lombiq.HelpfulExtensions.Extensions.Flows.Handlers;
+using Lombiq.HelpfulExtensions.Extensions.Flows.Models;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Display.ContentDisplay;
+using OrchardCore.ContentManagement.Handlers;
+using OrchardCore.DisplayManagement.Descriptors;
+using OrchardCore.Modules;
+using static Lombiq.HelpfulExtensions.FeatureIds;
+
+namespace Piedone.HelpfulExtensions.Extensions.Flows
+{
+    [Feature(Lombiq_HelpfulExtensions_Flows)]
+    public class Startup : StartupBase
+    {
+        override public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddScoped<IContentDisplayDriver, AdditionalStylingPartDisplay>();
+            services.AddScoped<IContentHandler, AdditionalStylingPartHandler>();

```

Comment:
These should be chained onto the part registration.

Reply:
Fun fact, you either use the part options feature and define Content*Part*Handler or you define a base ContentHandler for your part. Both are working and not using any obsolete features.
But you are right it'd be better with the options approach.

Reply:
Unfortunately, it won't work. As for the handler it's because that is not targeted to the part but it's targeted for the content item where we search for the Widget stereotype.
As for the driver it might be a bug (or not), the ContentItemDisplayCoordinator.BuildDisplayAsync (which is executed) only gathers the parts by the ContentTypePartDefinition and since this part is welded it won't be there so the driver will be completely ignored in the end.